client/src/components/Login/Login.jsx

```diff
@@ -1,45 +1,65 @@
-import style from './Login.module.css'
-import cx from 'classnames';
-import { Link, useNavigate } from 'react-router-dom';
-import severConnection from '../../services/dbServices';
-import { useEffect } from 'react';
+import style from "./Login.module.css";
+import cx from "classnames";
+import { Link, useNavigate } from "react-router-dom";
+import { useEffect } from "react";
 
 function Login({ getUserInfoAction, loginStatus }) {
+	useEffect(() => {
+		if (loginStatus === "in") navigate("/home");
```

Comment:
```suggestion
		if (loginStatus === LoginConstansts.logedIn) navigate("/home");
```
if you all the strings in a const file it will be much more easy to change and detect bugs